tox.ini

```diff
@@ -2,11 +2,15 @@
 envlist = py{37,36,35,34,27,py3,py}
 
 [testenv]
-deps = pytest-cov
-       py{37,36,35,27,py3,py}: pytest-timeout
-       py{34}: pytest-timeout<1.2.1
-       py{34}: pytest<3.3
-commands = py.test {posargs}
+deps =
+    py{37,36,35,27,py3,py}: pytest-cov
+    py{37,36,35,27,py3,py}: pytest-timeout
+    py{34}: pytest-timeout<1.2.1
+    py{34}: pytest<3.3
+    py{34}: pytest-cov==2.6.0
```

Comment:
These versions for Python 3.4 conflict with the `tests_require` in the `setup.py`.

Reply:
Should we remove `tests_require`? Same for all test stuff from `setup.py`. We can move `pytest` arguments to `setup.cfg`.
And for the testing itself, we use `tox`. If someone wants to test manually, it just launch `pytest`.

Reply:
FTR `pytest-cov` > 2.6.0 on Python 3.4.
I am wondering why we keep 3.4 in the Python versions we support. As the next version will drop 2.7, 3.2 and 3.3: 3.4 could be dropped too I think.

Reply:
The next version will drop 2.6, not 2.7. I'd avoid dropping any other version, since this is a library, not an application. I agree that probably most people aren't using Python 3.4, but the code does work on it, it's still maintained (in the "security fixes only" mode), and it's just the testing stuff here that seems to require some specifics. Libraries should always work on most versions, and three more lines in tox.ini won't hurt anyone. But we can make a plan to get rid from Python 3.4 later, I just think we should always try to maximize the "range" of environments `watchdog` can work on.

I agree we should remove `tests_require` and other test stuff from `setup.py`.

The `setup.cfg` is a file I would try to remove if the documentation wasn't configured on it. But since it does exist and it's useful, I liked the idea of putting the `pytest` arguments there.

Reply:
Agreed on all but for `setup.cfg` I would rather try to use only this file :)